src/components/Seo.tsx

```diff
@@ -16,6 +17,16 @@ export interface SeoProps {
   searchOrder?: number;
 }
 
+const deployedTranslations = [
+  'en',
+  // We'll add more languages when they have enough content.
+];
+
+function getDomain(languageCode: string): string {
+  const subdomain = languageCode === 'en' ? '' : languageCode + '.';
+  return subdomain + 'react.dev';
```

Comment:
We're going to assume this is where they canonically point — even before some translation's docs are stable enough to be published at the domain.